views/forms/RegisterForm/index.jsx

```diff
@@ -22,6 +22,7 @@ const RegistrationForm = ({ initialValues, onSubmit }) => {
     formState: { errors },
   } = useForm({
     defaultValues: initialValues,
+    mode: 'onChange',
```

Comment:
I don't think we want to validate on change because then the user is presented with validation messages as they type into the form. Are the validation errors showing up when an invalid form is submitted without this change? The issue was that they were not showing up at all.

Reply:
without  `mode: 'onChange'` yes the validation errors display after we submit the form. If there's a server error we display an error.  This is a screenshot of staging after submitting an incorrect form.

<img width="264" alt="Screenshot 2024-03-21 at 7 52 09 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/2206a24d-0f56-4358-b63b-127bed6915d5">

Reply:
ok then I think we can close this PR and move the ticket over to stakeholder review.